src/lib/stable/canister_methods/index.ts

```diff
@@ -6,50 +6,54 @@ import {
     CanisterMethodMode,
     executeAndReplyWithCandidSerde
 } from '../execute_with_candid_serde';
+import { QueryOptions } from './query';
+import { UpdateOptions } from './update';
 
 export interface ExportedCanisterClass {
     _azleCallbacks?: {
-        [key: string]: MethodType<ExportedCanisterClass, any[], any>;
+        [key: string]: (args?: Uint8Array) => Promise<void>;
     };
     _azleCanisterMethodIdlTypes?: { [key: string]: IDL.FuncClass };
     _azleCanisterMethodsIndex?: number;
     _azleInitAndPostUpgradeIdlTypes?: IDL.FuncClass[];
     _azleMethodMeta?: MethodMeta;
+    _azleShouldRegisterCanisterMethods?: boolean;
 }
 
-export type MethodType<This, Args extends any[], Return> = (
+export type MethodType<This, Args extends unknown[], Return> = (
     this: This,
     ...args: Args
 ) => Return;
 
-export type DecoratorFunction<This, Args extends any[], Return> = (
-    originalMethod: MethodType<This, Args, Return>,
-    context: ClassMethodDecoratorContext<This, MethodType<This, Args, Return>>
-) => MethodType<This, Args, Return>;
+export type DecoratorFunction<This, Args extends unknown[], Return> = (
+    originalMethod: OriginalMethod<This, Args, Return>,
+    context: Context<This, Args, Return>
+) => void;
```

Comment:
I presume that after investigating why inspect_message and heartbeat returned void while the other methods returned MethodType you discovered that all of them could return void?

Reply:
I am reviewing as if that assumption is true